certbot/certbot/_internal/client.py

```diff
@@ -470,8 +470,17 @@ def _get_order_and_authorizations(self, csr_pem: bytes,
         """
         if not self.acme:
             raise errors.Error("ACME client is not set.")
+
+        profile = None
+        available_profiles = self.acme.directory.meta.profiles
+        preferred_profile = self.config.preferred_profile
+        if self.config.required_profile is not None:
+            profile = self.config.required_profile
+        elif (preferred_profile is not None and available_profiles is not None and
+              preferred_profile in available_profiles):
```

Comment:
This feels pretty verbose. Is there a better or more Pythonic way to do it?

Reply:
If I understand truthiness in Python, maybe.... 
```
elif preferred_profile and available_profiles and preferred_profile in available_profiles:
```

Not sure how much error condition that leaves though in context, relying on Python truthiness. I am assuming "not-empty" instead of a specific value looking at the condition set here.